package.json

```diff
@@ -92,10 +89,10 @@
     "path-type": "^4.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.10.4",
-    "@babel/core": "^7.10.4",
-    "@babel/preset-env": "^7.10.4",
-    "@babel/preset-typescript": "^7.10.4",
+    "@babel/cli": "7.19.3",
+    "@babel/core": "7.20.2",
+    "@babel/preset-env": "7.20.2",
+    "@babel/preset-typescript": "7.18.6",
```

Comment:
Any reason why this switches to exact versioning?

Reply:
I suppose not.